### PR TITLE
tags updated in innovation record pages

### PR DESCRIPTION
**Pull request by Emanuel-Mariano-BJSS**

### CODE REVIEW

File: src/modules/stores/innovation/innovation.store.ts [hunk 1]
```diff
@@ -65,13 +65,14 @@ export class InnovationStore extends Store<InnovationModel> {
         sections: INNOVATION_SECTIONS.map(item => ({
           title: item.title,
           sections: item.sections.map(ss => {
-            const sectionState = response.sections.find(a => a.section === ss.id) || { status: 'UNKNOWN', actionStatus: '' };
+            const sectionState = response.sections.find(a => a.section === ss.id) || { status: 'UNKNOWN', actionStatus: '', actionCount: 1 };
```
Comment (BJSS-RicardoTavares) [inline src/modules/stores/innovation/innovation.store.ts:68]: Why 1 and not zero?

File: src/modules/stores/innovation/innovation.models.ts [hunk 2]
```diff
@@ -44,6 +44,7 @@ export type sectionType = {
   section: InnovationSectionEnum;
   status: keyof typeof INNOVATION_SECTION_STATUS;
   actionStatus: keyof typeof INNOVATION_SECTION_ACTION_STATUS;
+  actionCount: number;
```
Comment (BJSS-RicardoTavares) [inline src/modules/stores/innovation/innovation.models.ts:47]: I believe that this change produces undesirable effects (in terms of typing's).

Call me to discuss this please...

File: src/modules/stores/innovation/innovation.store.ts (hunk 1 above)
Comment (Emanuel-Mariano-BJSS) [reply src/modules/stores/innovation/innovation.store.ts]: Yes, it should be zero. Fixed

File: src/modules/stores/innovation/innovation.models.ts (hunk 2 above)
Comment (Emanuel-Mariano-BJSS) [reply src/modules/stores/innovation/innovation.models.ts]: Fixed DTO

**Review by BJSS-RicardoTavares — APPROVED** [review]

**Pull request merged**